src/utils/Draft.js

```diff
@@ -40,7 +40,7 @@ export const addSeen = (seen, cards, synergies) => {
         const combStr = comb.join('');
         if (COLOR_INCLUSION_MAP[combStr][colorsStr]) {
           for (const { index } of seen.cards[combStr]) {
-            if (synergyMatrix[index][card.index] === null) {
+            if (synergyMatrix[index][card.index] === null && synergies) {
```

Comment:
Need to set the value to zero so we don't add null below

Reply:
Seems like I accidentally committed this workaround to get the draft view working at master now. I'll just revert that as it was never supposed to end up in this PR anyways, good catch.